internal/cmd/egctl/translate.go

```diff
@@ -180,32 +180,55 @@ func translateFromGatewayAPIToXds(w io.Writer, inBytes []byte) error {
 }
 
 // printXds prints the xDS Output
-func printXds(w io.Writer, key string, tCtx *xds_types.ResourceVersionTable) error {
-	fmt.Fprintf(w, "\nKey: %s\n", key)
+func printXds(w io.Writer, key string, tCtx *xds_types.ResourceVersionTable, output string) error {
 	bootsrapYAML, err := infra.GetRenderedBootstrapConfig()
```

Comment:
can we reuse https://github.com/envoyproxy/go-control-plane/blob/6d01c4d91f2fb76887be2e7b155b75c63d2458e5/envoy/admin/v2alpha/config_dump.pb.go#L162?

Reply:
+1 to @zirain's suggestion of leveraging [configDump.Configs](https://www.envoyproxy.io/docs/envoy/latest/api-v3/admin/v3/config_dump.proto#admin-v3-configdump)